crates/staked-relayers/src/lib.rs

```diff
@@ -128,8 +105,21 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        /// Upgrade the runtime depending on the current `StorageVersion`.
         fn on_runtime_upgrade() -> Weight {
-            Self::_on_runtime_upgrade().expect("runtime upgrade failed");
+            use frame_support::{migration::StorageKeyIterator, Blake2_128Concat};
+
+            StorageKeyIterator::<T::AccountId, StakedRelayer<DOT<T>, T::BlockNumber>, Blake2_128Concat>::new(<Stakes<T>>::module_prefix(), b"ActiveStakedRelayers")
```

Comment:
Please double check

Reply:
Seems fine to me, but maybe we should add a small unit test?

Reply:
How would you write a unit test for this?

Reply:
Insert the previous key / value formats into storage and check the iterator correctly updates.